nixos/modules/services/x11/desktop-managers/xfce4-13.nix

```diff
@@ -0,0 +1,142 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I still think that duplicating this module is a bad idea. Xfce is unlikely to radically change, and we would be duplicating a lot of logic, especially if we were ever to implement declarative config we've discussed earlier.

Reply:
No problem.
My main goal about 4.13 is to fix some minor bugs (like the clocks that after the computer woke up shows the time when computer went to the sleep mode). Upstream won't accept patches against 4.12 and we do not have working 4.13.
If de-duplicating the modules speeds up the whole work a bit, I do not mind.